bin/hyperstream

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env node
 
-var hyperstream = require('..')
-var minimist = require('minimist')
-var low = require('last-one-wins')
-var util = require('../lib/util')
+lethyperstream = require('..')
```

Comment:
is it just me, or there's no space after "let"?

Reply:
You are right early on I did a find and replace that went wrong but I never even looked in the bin folder as I assumed it was something that had been built/transpiled I have corrected this now.

I will take a look at converting this file to TS as well, 

I think this PR is still a WIP I didnt want to dictate style/linting and I am really looking for some feedback on whether or not this is the direction everyone imagined this conversion to TS going.

Thanks